core/src/i18n.rs

```diff
@@ -10,10 +10,7 @@ static_loader! {
 }
 
 pub fn core_text(language: &LanguageIdentifier, id: &str) -> String {
-    TEXTS.lookup(language, id).unwrap_or_else(|| {
```

Comment:
looking at how the non-fallible function works, are you sure we don't actually want a `.try_lookup().unwrap_or_else()`?

Reply:
Fixed